docs/source/conf.py

```diff
@@ -77,34 +77,27 @@
 
 # The code below will cut version displayed in the dropdown like this:
 # tags like v0.1.0 = > 0.1
-# branch like release/0.1 => 0.1
 # main will remain main
 # if not set will fail back to main
 # the version varible is used in layout.html: https://github.com/pytorch/executorch/blob/main/docs/source/_templates/layout.html#L29
 if et_version_docs:
-    # Check if starts with release/ and set the version to the number after slash
-    if et_version_docs.startswith("release/"):
+    if et_version_docs.startswith("refs/tags/v"):
+        version = ".".join(
+            et_version_docs.split("/")[-1].split("-")[0].lstrip("v").split(".")[:2]
+        )
```

Comment:
Please add a comment describing the format that this expects to see. E.g., "refs/tags/v1.2.3-rc4", with or without the `-rcN` part.